html/pfappserver/root/admin/radiuslog.tt

```diff
@@ -0,0 +1,55 @@
+[%  jsFiles = [
```

Comment:
Shouldn't this file be named auditing.tt even if it only contains the RADIUS audit log now.

My understanding is that more reports will join that section shortly (admin auditing) and will share the page as a base.